seleniumwire/thirdparty/mitmproxy/connections.py

```diff
@@ -302,28 +307,32 @@ def finish(self):
 
 
 class SocksServerConnection(ServerConnection):
-
     def __init__(self, socks_config, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.socks_config = socks_config
 
+    def getaddrinfo(self, host, port, *args, **kwargs):
+        if self.socks_config.scheme == "socks5h":
+            return [(socket.AF_INET, socket.SOCK_STREAM, 6, "", (host, port))]
+        return super().getaddrinfo(*args, **kwargs)
```

Comment:
Think this may need to explicitly pass the `host` and `port` as they're not captured by `args`, i.e.
```python
return super().getaddrinfo(host, port, *args, **kwargs)
```

Reply:
Fixed in bf755b3